tests/test_oai.py

```diff
@@ -0,0 +1,233 @@
+        '          <givenName>Lars Holm</givenName>\n'
+        '          <familyName>Nielsen</familyName>\n'
+        '          <nameIdentifier nameIdentifierScheme="ORCID">0000-0001-8135-3489</nameIdentifier>\n'  # noqa
+        '          <affiliation affiliationIdentifier="01ggx4157" affiliationIdentifierScheme="ROR">CERN</affiliation>\n'  # noqa
+        '        </contributor>\n'
+        '      </contributors>\n'
+        '      <dates>\n'
+        '        <date dateType="Issued">2018/2020-09</date>\n'
+        '        <date dateType="Other" dateInformation="A date">1939/1945</date>\n'  # noqa
+        '      </dates>\n'
+        '      <language>dan</language>\n'
+        '      <resourceType resourceTypeGeneral="Image">Photo</resourceType>\n'  # noqa
+        '      <relatedIdentifiers>\n'
+        '        <relatedIdentifier relatedIdentifierType="DOI" relationType="IsCitedBy" resourceTypeGeneral="Dataset">10.1234/foo.bar</relatedIdentifier>\n'  # noqa
+        '      </relatedIdentifiers>\n'
+        '      <sizes>\n'
+        '        <size>11 pages</size>\n'
+        '      </sizes>\n'
+        '      <formats>\n'
+        '        <format>application/pdf</format>\n'
+        '      </formats>\n'
+        '      <version>v1.0</version>\n'
+        '      <rightsList>\n'
+        '        <rights rightsURI="https://customlicense.org/licenses/by/4.0/">A custom license</rights>\n'  # noqa
+        '        <rights rightsURI="https://creativecommons.org/licenses/by/4.0/legalcode" rightsIdentifierScheme="spdx" rightsIdentifier="cc-by-4.0">Creative Commons Attribution 4.0 International</rights>\n'  # noqa
+        '      </rightsList>\n'
+        '      <descriptions>\n'
+        '        <description descriptionType="Abstract">A description with HTML tags</description>\n'  # noqa
+        '        <description descriptionType="Methods" xml:lang="eng">Bla bla bla</description>\n'  # noqa
+        '      </descriptions>\n'
+        '      <geoLocations>\n'
+        '        <geoLocation>\n'
+        '          <geoLocationPlace>test location place</geoLocationPlace>\n'
+        '          <geoLocationPoint>\n'
+        '            <pointLongitude>-60.63932</pointLongitude>\n'
+        '            <pointLatitude>-32.94682</pointLatitude>\n'
+        '          </geoLocationPoint>\n'
+        '        </geoLocation>\n'
+        '      </geoLocations>\n'
+        '      <fundingReferences>\n'
+        '        <fundingReference>\n'
+        '          <funderName>European Commission</funderName>\n'
+        '          <funderIdentifier funderIdentifierType="ROR">1234</funderIdentifier>\n'  # noqa
+        '          <awardNumber>246686</awardNumber>\n'
+        '          <awardTitle>OpenAIRE</awardTitle>\n'
+        '        </fundingReference>\n'
+        '      </fundingReferences>\n'
+        '    </resource>\n'
+        '  </payload>\n'
+        '</oai_datacite>\n'
```

Comment:
@slint is this wrapping correct?

Reply:
Yes, see also an [example from Zenodo](https://zenodo.org/oai2d?verb=GetRecord&metadataPrefix=oai_datacite&identifier=oai:zenodo.org:1228465)